### PR TITLE
User form to get Client Credentials.

### PR DESCRIPTION
**Pull request by jyokush**
userForm.Design.vb  file is compiler generated code when i made the user form manually.

### CODE REVIEW

File: userForm.vb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            'End If
+            lines = hittingEndPoint.parseTheFile()
+            'If the data list count item Is Not zero Then all Api In the file Is Not supported.
+            If lines.Count <> 0 Then
+                Dim userControlObject As Object = Nothing
+                Dim userControlToolWindow As UserControlToolWindow
+                Try
+                    'Create the Tool Window to show the result.
+                    userControlToolWindow = New UserControlToolWindow()
+                    CreateToolWindow("Report of API used in this project", "B9055551-73E5-4507-AB69-19FF25D00F2B", userControlToolWindow)
```
Comment (monikadhok) [inline userForm.vb]: "Report of API used in this project" >> API Summary

Comment (monikadhok) [inline userForm.vb]: What is this id B9055551-73E5-4507-AB69-19FF25D00F2B

File: userForm.vb [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    End Sub
+
+    'Click event of submit button.
+    Private Sub Submit_Click(sender As Object, e As EventArgs) Handles Submit.Click
+        'Get the value in the text box of the form.
+        _getClientId = ClientIdInput.Text
+        _getClientSecretId = ClientSecretIdInput.Text
+        'If the client Id and client secret id is not empty. 
+        If _getClientId IsNot Nothing And _getClientSecretId IsNot Nothing Then
+            ''Make a copy of the file active in excel.
```
Comment (monikadhok) [inline userForm.vb]: What is this commented code

File: userForm.vb (hunk 1 above)
Comment (jyokush) [reply userForm.vb]: Done

Comment (jyokush) [reply userForm.vb]: Done

File: userForm.vb (hunk 2 above)
Comment (jyokush) [reply userForm.vb]: Done

**Review by monikadhok — APPROVED** [review]

**Pull request merged**